bundles/org.openhab.binding.nuki/src/main/java/org/openhab/binding/nuki/internal/handler/NukiSmartLockHandler.java

```diff
@@ -248,6 +255,10 @@ private boolean handleResponse(NukiBaseResponse nukiBaseResponse, String channel
         if (channelLockState != null) {
             updateState(channelLockState.getUID(), new DecimalType(NukiBindingConstants.LOCK_STATES_UNDEFINED));
         }
+        Channel channelDoorState = thing.getChannel(NukiBindingConstants.CHANNEL_SMARTLOCK_DOOR_STATE);
+        if (channelDoorState != null) {
+            updateState(channelDoorState.getUID(), new DecimalType(NukiBindingConstants.DOORSENSOR_STATES_UNKNOWN));
```

Comment:
See other comment about channel item-type:
```suggestion
            updateState(channelDoorState.getUID(), new StringType(NukiBindingConstants.DOORSENSOR_STATES_UNKNOWN));
```